packages/tasks/src/tokenizer-data.ts

```diff
@@ -0,0 +1,24 @@
+export const SPECIAL_TOKENS_ATTRIBUTES = [
```

Comment:
**TODO (in follow-up PR):** use this list in moon-landing to extract special tokens from `tokenizer_config.json` file. Currently [only bos and eos are parsed](https://github.com/huggingface/moon-landing/blob/4d7c7718f63b37e828a467d415dc7df62d9b07c5/server/repo_types/ModelInfo.ts#L88).

Reply:
For reference, the list is taken from [transformers](https://github.com/huggingface/transformers/blob/3de6a6b4936229e3b4467dd7de1c24f2fae64528/src/transformers/tokenization_utils_base.py#L823-L832). Also, we should get that merged before this one.